src/sidebar/components/tag-editor.js

```diff
@@ -61,7 +91,7 @@ function TagEditor({ onEditTags, tags: tagsService, tagList }) {
    * reset the activeItem and open the AutocompleteList
    */
   const updateSuggestions = () => {
-    const value = inputEl.current.value.trim();
+    const value = inputEl.current ? inputEl.current.value.trim() : '';
```

Comment:
In a number of cases, I got complaints about `current` being undefined. Sure, before and after a component mounts, its possible that this value is "undefined," however in practice I don't see this actually happening unless we have callbacks firing after a component unmounts. Therefore, I don't feel it's worthwhile adding a test for this condition. Frankly, I'm not even sure how to go about writing such a test.

Reply:
There will be scenarios where we know something cannot be null/undefined for reasons that require specific domain knowledge. In TypeScript there is an "I know what I'm doing" operator (`!`) for this: `inputEl.current!.value.trim()`. In JS unfortunately you have to be more verbose, eg. `/** @type {HTMLInputElement} */ (input.current).value.trim()`, or you could create a variable if you were going to reference the object multiple times. [Related upstream issue](https://github.com/microsoft/TypeScript/issues/23405).

Reply:
roger, casting seems appropriate here